packages/core/src/components/InputGroup/InputGroupText.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <pf-input-group-item v-bind="ouiaProps" box>
-    <component :is="component" :class="styles.inputGroupText">
+    <component :is="component" :class="styles.inputGroupText,{
+        [styles.modifiers.plain]: plain
+      }">
```

Comment:
This should be an array

```suggestion
    <component :is="component" :class="[styles.inputGroupText, {
        [styles.modifiers.plain]: plain
      }]">
```